metadata-ingestion/tests/integration/kafka-connect/kafka_connect_mces_golden.json

```diff
@@ -1103,6 +1103,21 @@
         "runId": "kafka-connect-run"
     }
 },
+{
+    "entityType": "dataJob",
+    "entityUrn": "urn:li:dataJob:(urn:li:dataFlow:(kafka-connect,postgres_source,PROD),postgres1.postgres.public.member)",
+    "changeType": "UPSERT",
+    "aspectName": "status",
+    "aspect": {
+        "json": {
+            "removed": false
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1635166800000,
+        "runId": "kafka-connect-run"
+    }
+},
```

Comment:
this status aspect is being emitted twice. Any reason why this is being emitted twice now?

Reply:
@anshbansal let me check, and if possible remove the duplicate aspect